src/lib.rs

```diff
@@ -73,24 +71,36 @@ impl InlinerBuilder {
         self
     }
 
-    /// Parse the source code in `src_file` and return a `syn::File` that has all modules
+    /// Parse the source code in `src_file` and return an `InliningResult` that has all modules
     /// recursively inlined.
     pub fn parse_and_inline_modules(&self, src_file: &Path) -> Result<InliningResult, Error> {
-        self.parse_internal(src_file, FsResolver::default())
+        // Rust seems to not be able to resolve callback lifetimes properly with an inline
```

Comment:
I ran into this problem before; I think you have to pass it as `&impl` or something and then it works.

Reply:
Ahh, I couldn't figure out what you meant but it looked like simply annotating one of the arguments with the type worked fine.